webapp/src/websocket.tsx

```diff
@@ -0,0 +1,12 @@
+import {Dispatch} from 'redux';
+
+import {GetStateFunc} from 'mattermost-redux/types/actions';
+import {WebSocketMessage} from 'mattermost-redux/types/websocket';
+
+import {setNoteValue} from 'redux_connectors';
+
+export function handleWebsocketNoteUpdate(_: GetStateFunc, dispatch: Dispatch) {
+    return async (msg: WebSocketMessage<{note: string}>) => {
+        dispatch(setNoteValue(msg.broadcast.channel_id, msg.data.note));
```

Comment:
This might be interesting with >1 person editing the same note. :)

Reply:
Yes. Ideally I want to implement a better system. If people actually run into this I will be honored and will promise to fix it.